### PR TITLE
Remove autopilot testdrive

### PR DESCRIPTION
**Pull request by pmahnke**
## Done

* We are getting rid of the Autopilot testdrive, this PR removes it from the site
 * Removed references to the testdrive from cloud section
 * Simplified /download/cloud to have the single page with instructions
 * Removed older instructions pages and created redirects
 * Updated other references and links to instructions

## QA

[demo site](http://www.ubuntu.com-remove-autopilot-testdrive.demo.haus/)

1. go to /download/cloud [copy doc](https://docs.google.com/document/d/1y3uaLNn7dxZKcLq2OmVJ96o3NOqn4OTJflty2GmxV9I/edit) and see that it works and matches the copy doc
2. go to /cloud [copy doc](https://docs.google.com/document/d/1mQjVXDcoP8ChOzRdmuI55BdXhbURD53qBNc2bgJhfhg/edit)  and see that it works and matches the copy doc
3. go to /cloud/openstack [copy doc](https://docs.google.com/document/d/12z6QEGpqMVbNPJOd5aUfn0VnXQ0Sa5YriEBLA7VGNwI/edit) and see that it works and matches the copy doc
4. go to /cloud/openstack/autopilot [copy doc](https://docs.google.com/document/d/1VmFtp_gEYvioRgGXbiQZpVGBhlJoIZAirLk6IOVUjUE/edit) and see that it works and matches the copy doc
5. see that the ebook download on /download/cloud still works
6. see that the redirects work (download/cloud/install-autopilot-testdrive/ > /download/cloud & download/cloud/install-openstack-with-autopilot/ > /download/cloud)

## Issue / Card

[trello card](https://trello.com/c/9MugT9N6)

### CODE REVIEW

**Comment by pmahnke** [comment]
<a href="https:&#x2F;&#x2F;trello.com&#x2F;c&#x2F;YS9OQxzc&#x2F;1622-design-review-autopilot-testdrive-removal"> design review autopilot testdrive removal</a>

File: templates/cloud/openstack/autopilot.html [hunk 1]
```diff
@@ -55,100 +55,93 @@
...
-                Get Autopilot for your hardware
+        <p>
+            <a class="button--primary" href="/download/cloud">
+            Get Autopilot
             </a>
+        </p>
+    </div>
+
+    <div class="six-col last-col">
+        <p><b>Requirements for Autopilot:</b></p>
```
Comment (deadlight) [inline templates/cloud/openstack/autopilot.html]: I know that you didn't introduce this but could we replace these with something nicer. <h4> looks good and is better, semantically, for a header

File: templates/cloud/openstack/autopilot.html [hunk 2]
```diff
@@ -55,100 +55,93 @@
     <h2>Build an OpenStack cloud with Autopilot</h2>
 
     <div class="six-col">
-        <h3>Install Autopilot on your own servers</h3>
         <p><b>Experience OpenStack at its best</b></p>
```
Comment (deadlight) [inline templates/cloud/openstack/autopilot.html]: <h4> for this one too

**Comment by deadlight** [comment]
Looks good to me.

One small fix mentioned in the code; fix that and feel free to self-approve the code.

**Comment by pmahnke** [comment]
@deadlight I make the changes you required.

**Pull request merged**